libs/@local/graph/postgres-store/src/store/postgres/seed_policies.rs

```diff
@@ -0,0 +1,332 @@
+                filters: create_version_filters(
+                    base_url!("https://hash.ai/@h/types/entity-type/hash-instance/"),
+                    1,
+                )
+                .chain(create_version_filters(
+                    base_url!("https://hash.ai/@h/types/entity-type/user/"),
+                    6,
+                ))
+                .chain(create_version_filters(
+                    base_url!("https://hash.ai/@h/types/entity-type/organization/"),
+                    2,
+                ))
+                .collect(),
+            },
+        })),
+    });
+
+    let authenticated_actor_policies = [ActorType::User, ActorType::Machine, ActorType::Ai]
+        .into_iter()
+        .map(|actor_type| PolicyCreationParams {
+            name: Some("authenticated-view-entity".to_owned()),
+            effect: Effect::Permit,
+            principal: Some(PrincipalConstraint::ActorType { actor_type }),
+            actions: vec![ActionName::ViewEntity],
+            resource: Some(ResourceConstraint::Entity(EntityResourceConstraint::Any {
+                filter: EntityResourceFilter::Any {
+                    filters: create_version_filters(
+                        base_url!("https://hash.ai/@h/types/entity-type/machine/"),
+                        2,
+                    )
+                    .chain(create_version_filters(
+                        base_url!("https://hash.ai/@h/types/entity-type/service-feature/"),
+                        1,
+                    ))
+                    .chain(create_version_filters(
+                        base_url!("https://hash.ai/@h/types/entity-type/is-member-of/"),
+                        1,
+                    ))
+                    .collect(),
+                },
+            })),
+        });
+
+    public_policies.chain(authenticated_actor_policies)
+}
+
+pub(crate) fn global_policies() -> impl Iterator<Item = PolicyCreationParams> {
+    global_instantiate_policies().chain(global_view_entity_policies())
+}
+
+fn web_view_entity_policies(role: &WebRole) -> impl Iterator<Item = PolicyCreationParams> {
+    iter::once(PolicyCreationParams {
+        name: Some("default-web-view-entity".to_owned()),
+        effect: Effect::Permit,
+        principal: Some(PrincipalConstraint::Role {
+            role: RoleId::Web(role.id),
+            actor_type: None,
+        }),
+        actions: vec![ActionName::ViewEntity],
+        resource: Some(ResourceConstraint::Entity(EntityResourceConstraint::Web {
+            web_id: role.web_id,
+            filter: EntityResourceFilter::Not {
+                filter: Box::new(EntityResourceFilter::Any {
+                    filters: create_version_filters(
+                        base_url!("https://hash.ai/@h/types/entity-type/user-secret/"),
+                        1,
+                    )
+                    .chain(create_version_filters(
+                        base_url!("https://hash.ai/@h/types/entity-type/usage-record/"),
```

Comment:
Not super important as we'll make these not entities at some point, but web _admins_ should be able to view these

Reply:
I actually just implemented this in a follow-up 😄